pyvista/demos/logo.py

```diff
@@ -41,7 +42,9 @@ def atomize(grid, shift_fac=0.1, scale=0.9):
 
 def text_3d(string, depth=0.5):
     """Create 3D text."""
-    vec_text = _vtk.vtkVectorText()
+    from vtkmodules.vtkRenderingFreeType import vtkVectorText
```

Comment:
This is because the import at the top was changed to `from pyvista.core import _vtk_core as _vtk` as the rest of the imports in this file from `_vtk` are only using core-features (no plotting/libGL-dependednt modules). Since `vtkVectorText` comes from `vtkRenderingFreeType` it cannot be in the `_vtk_core` module and thus I've changed this in a few places to import at "runtime" within the called function. Another example is `pyvista.Text3D`